### ISSUE TITLE
Terminal Crashing when pasting code from VS Code

### ISSUE THREAD

**Issue by DevTechSolutions** [microsoft/terminal#14081]
### Windows Terminal version

1.14.2281.0

### Windows build number

10.0.19044.0

### Other Software

VS Code
Version: 1.71.2 (user setup)
Commit: 74b1f979648cc44d385a2286793c226e611f59e7
Date: 2022-09-14T21:03:37.738Z
Electron: 19.0.12
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19044
Sandboxed: No

### Steps to reproduce

Open VS Code
Open Windows Terminal
Select text in VS Code using right-click Copy
Right-Click to paste into Terminal
Terminal window crashes with no error message

### Expected Behavior

The text to be pasted into the terminal window.

From my testing this only seems to occur when copy/paste from VS Code. Pasting text from Word or a Browser works as expected.

### Actual Behavior

The application closes with no error message.

**Comment by carlos-zamora** [comment]
Doesn't repro with 1.14.2282 (Windows 11 version)

We need to verify this with the Windows 10 version. Throwing it in the current milestone for someone to investigate.

**Comment by lhecker** [comment]
I wrote "foobar" in VS Code, right-click copied it and right-click pasted it into Terminal 1.15 without crashes. I then opened a 100MB text file copied and pasted it the same way. Apart from gobbling up 2GB of memory and taking ages to complete, nothing happened there either.

@DevTechSolutions If this issue still happens for you, I believe we'd need more details: What shell and shell version are you using? What text reproduces the issue? Even plain text without any plugins loaded in VS Code?
